sdk/appconfiguration/app-configuration/package.json

```diff
@@ -47,8 +47,8 @@
     "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
     "build:samples": "echo Obsolete.",
     "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
-    "clean": "rimraf dist dist-esm dist-browser test-dist test-browser types *.tgz *.log",
-    "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 test-dist/index.js --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "clean": "rimraf dist dist-esm dist-browser dist-test types *.tgz *.log",
```

Comment:
How about this?
```suggestion
    "clean": "rimraf dist* types *.tgz *.log",
```

Reply:
(Everywhere)

Reply:
We can do it either way, it works. Sometimes it's better to list out individual output folders so we know exactly what is being cleaned. But I don't have a preference. @HarshaNalluru

Reply:
Thinking more about this, I feel "dist* might be a bad choice, v/s "dist dist-*" that the communication package originally had, as "dist*" might start deleting unrelated files like "distributing.md" or "distribution-tracing.csv" We don't currently have such examples in our repo, but that can be a possibility, right? @HarshaNalluru